sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidget.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.maps.renderer.widget
+
+import android.graphics.Bitmap
+
+open class BitmapWidget(
+  bitmap: Bitmap,
+  position: WidgetPosition = WidgetPosition.BOTTOM_LEFT,
+  marginX: Float = 0f,
+  marginY: Float = 0f,
+) : Widget() {
+  override val renderer = BitmapWidgetRendererImpl(
```

Comment:
@mapbox/maps-android I've extracted logic for rendering bitmap to a separate entity, and made it internal to hide from users. However, this blocks disallows inheritance in android-auto module. Thus BitmapWidget and corresponding BitmapWidgetRenderer now lives here, in the main sdk module. WDYT about it? Guess binary size will be slightly bigger.

Reply:
that is fine, conceptually we have been hosting them in sdk-base but that was more for plugins. For extensions, that have a dependency on sdk module this is ok. I'm not expecting too much binary size addition.

Reply:
@tobrun another thing to note is that `class Widget`, `class BitmapWidget`, and `mapView.addWidget` are new public API's. Guess I'll also add `@MapboxExperimental` to Widget/BitmapWidget but question is about the public API updates - are we fully ok with it? 
Since android-auto extension that is the only user is NOT yet part of the V10 yet - would we like to hide it somehow? Suppose, at least `mapView.addWidget` can be made internal and discovered with reflection, for example.

Reply:
Adding MapboxExperimental to `addWidget` is acceptable to serve android-auto